validation-test/stdlib/String.swift

```diff
@@ -999,6 +1000,30 @@ StringTests.test(
 #endif
 }
 
+StringTests.test("String._fastCStringContents()")
+  .skip(.nativeRuntime("_NSContiguousString undefined without _runtime(_ObjC)"))
+  .code {
+#if _runtime(_ObjC)
```

Comment:
I don't think it needs to be a part of this PR, just as long as we don't add more of this pattern.